web/secure_css_wrapper.php

```diff
@@ -29,6 +29,7 @@
 $content = file_get_contents($fileUri);
 
 // replace links to http://www.wikidot.com/... and to the current host.
+# XXX mix-up between http and https?
 $content = preg_replace(';url\(http://([a-z0-9\-]+)\.{$URL_DOMAIN_PREG}/;i', 'url(https://\\1.{$URL_DOMAIN}/', $content);
```

Comment:
```suggestion
$content = preg_replace('|url\(http://([a-z0-9\-]+)\.{$URL_DOMAIN_PREG}/|i', 'url(https://\\1.{$URL_DOMAIN}/', $content);
```

Reply:
I think I'd prefer consistency here - forward slash if possible, semi colon otherwise. Unless you'd want to make a more general sweep over the regexes to replace all semi colon delimited regexes with pipes, which I'd be fine with, changing just this one seems odd